tests/unit-core/test-abort.c

```diff
@@ -118,15 +118,15 @@ main (void)
   TEST_ASSERT (jerry_value_is_abort (value));
   TEST_ASSERT (jerry_value_is_error (value));
 
-  jerry_value_set_error_flag (&value);
-  TEST_ASSERT (!jerry_value_is_abort (value));
-  TEST_ASSERT (jerry_value_is_error (value));
+  jerry_value_t error = jerry_create_error_from_value (value, true);
+  TEST_ASSERT (!jerry_value_is_abort (error));
+  TEST_ASSERT (jerry_value_is_error (error));
```

Comment:
I would prefer:
`value = jerry_create_error_from_value (value, true);`

Otherwise we need to introduce an `abort` identifier later, and that is a symbol clash with a libc function.